pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -48,24 +48,31 @@ func (c *clientHTTP) downloadFile() error {
 	}
 
 	if c.saveFilePath == "" {
-		return errors.New("saveFilePath must be specificed")
+		return httpError404, errors.New("saveFilePath must be specificed")
 	}
 
 	client := &http.Client{Transport: tr}
 	resp, err := client.Get(c.urlAddr)
+
+	switch resp.StatusCode {
+	case httpError404:
+		msg := fmt.Sprintf("http response 404 for: %s", c.urlAddr)
+		return resp.StatusCode, errors.New(msg)
```

Comment:
why the msg and return errors.New(msg), maybe just use `https://godoc.org/github.com/pkg/errors#Errorf`

Reply:
thanks!